variables.mk

```diff
@@ -65,8 +79,8 @@ REBAR_FIRRTL_DIR = $(base_dir)/tools/firrtl
 long_name = $(PROJECT).$(MODEL).$(CONFIG)
 
 # if building from rocketchip, override the long_name to match what they expect
-ifeq ($(SBT_PROJECT),rebarrocketchip)
-	long_name=$(PROJECT).$(CONFIG)
+ifeq ($(PROJECT),freechips.rocketchip.system)
```

Comment:
Hwacha also uses this version of `long_name`

Reply:
Ok so it sounds like there needs to be another `ifeq` then.